### PR TITLE
UHF-202 Article

### PR DESCRIPTION
**Pull request by khalima**
How to test: 
- Check instructions from https://github.com/City-of-Helsinki/drupal-hdbt/pull/35
- When the one above is approved, you may approve this one as well

### CODE REVIEW

File: src/scss/_common.scss [hunk 1]
```diff
@@ -207,3 +207,71 @@ body {
 .node-form .field--name-field-hero thead .field-label {
   display: none;
 }
+
+// Select2 to look and feel more Gin-like.
+.select2-container {
+  .select2-selection {
+    border: 1px solid var(--colorGinFormElementBorder);
+    border-radius: 6px;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, .25);
```
Comment (teroelonen) [inline src/scss/_common.scss:216]: You can use variable inside the rgba as well so like ($color-black, .25).

Comment (teroelonen) [reply src/scss/_common.scss:216]: But this is admin theme so you can leave it if it doesn't bother you.

**Review by teroelonen — APPROVED** [review]
Some very very small but approve anyway!

File: src/scss/_common.scss (hunk 1 above)
Comment (khalima) [reply src/scss/_common.scss:216]: TBH That's a copy-paste and I feel the shame. 😁

**Pull request merged**